src/app/search/state/search.service.ts

```diff
@@ -83,80 +83,87 @@ export class SearchService {
-    { friendlyName: 'Source Control', esName: 'source_control_provider.keyword', initiallyExpanded: true },
-    { friendlyName: 'Namespace', esName: 'namespace', initiallyExpanded: true },
-    { friendlyName: 'Organization', esName: 'organization', initiallyExpanded: true },
-    { friendlyName: 'Labels', esName: 'labels.value.keyword', initiallyExpanded: false },
-    {
-      friendlyName: 'Private Access',
-      esName: 'private_access',
-      tooltip: "A private tool requires authentication to view on Docker's registry website and to pull the Docker image.",
-      initiallyExpanded: false,
-      exclusive: true,
-    },
-    {
-      friendlyName: 'Verified Source',
-      esName: SearchFields.VERIFIED_SOURCE,
-      tooltip: 'Indicates which party performed the verification process on a tool or workflow.',
-      initiallyExpanded: false,
-    },
-    {
-      friendlyName: 'Verified Platforms',
-      esName: 'verified_platforms.keyword',
-      tooltip: 'Indicates which platform a tool or workflow (at least one version) was successfully run on.',
-      initiallyExpanded: false,
-    },
-    { friendlyName: 'Input File Formats', esName: 'input_file_formats.value.keyword', initiallyExpanded: false },
-    { friendlyName: 'Output File Formats', esName: 'output_file_formats.value.keyword', initiallyExpanded: false },
-    {
-      friendlyName: 'Verified',
-      esName: 'verified',
-      tooltip: 'Indicates that at least one version of a tool or workflow has been successfully run by our team or an outside party.',
-      initiallyExpanded: true,
-      exclusive: true,
-    },
-    {
-      friendlyName: 'Has Checker Workflow',
-      esName: 'has_checker',
-      tooltip:
-        'Checker workflows are additional workflows you can associate with a tool or workflow to ensure ' +
-        'that, when given some inputs, it produces the expected outputs on a different platform other than the one it was developed on.',
-      initiallyExpanded: false,
-      exclusive: true,
-    },
-    {
-      friendlyName: 'Open Data',
-      esName: 'openData',
-      tooltip:
-        'Indicates whether an entry can be run with no additional access permissions, potentially via an included test parameter file referencing open data.',
-      initiallyExpanded: false,
-      exclusive: true,
-    },
-  ];
 
-  /**
-   * These are the terms which use "must" filters
-   * Example: Results returned can be private or public but never both
-   * @memberof SearchService
-   */
-  public exclusiveFilters = this.orderedFacetInfos.filter((facetInfo) => facetInfo.exclusive).map((facetInfo) => facetInfo.esName);
+  private getOrderedFacetInfos(tabIndex: number): Array<FacetInfo> {
+    return [
+      { friendlyName: 'Category', esName: 'categories.name.keyword', initiallyExpanded: true },
+      {
+        friendlyName: tabIndex === SearchService.NOTEBOOKS_TAB_INDEX ? 'Format' : 'Language',
+        esName: 'descriptorType',
```

Comment:
Hypothetical problem: I could see wanting to have language expanded for tools/workflows, but not expanded for Notebooks, since we currently only support one format for notebooks. But the way the expansion saving logic looks like it works, keyed off `esName`, it looks like they would clobber each other. No need to change, just noting -- nobody will probably even notice this.

Reply:
The initial "expanded" setting will be for whatever entry type is searched first, and that will persist across different entry types, until changed by the user.  Not 100% ideal, but probably works good enough.  If we ever decide it doesn't, we can add another key (the entry type) to the save/restore logic.